### ISSUE TITLE
Cannot send KeyEvent with Appium v1.3.5

### ISSUE THREAD

**Issue by hectorv** [appium/appium#4549]
I've been using key code 82 to get the Menu in our app and just now that I tried to run our tests on Appium v1.3.5 I noticed that it doesn't work anymore. None of these:

```python
self.driver.press_keycode(82)
self.driver.keyevent(82)
```

```
2015-02-17 17:01:58:445 - info: --> POST /wd/hub/session/7192f3b3-7680-4fec-9d3d-09eb2b916930/appium/device/press_keycode {"sessionId":"7192f3b3-7680-4fec-9d3d-09eb2b916930","keycode":82}
2015-02-17 17:01:58:445 - info: [debug] Pushing command to appium work queue: ["pressKeyCode",{"keycode":82,"metastate":null}]
2015-02-17 17:01:58:447 - info: [debug] [BOOTSTRAP] [debug] Got data from client: {"cmd":"action","action":"pressKeyCode","params":{"keycode":82,"metastate":null}}
2015-02-17 17:01:58:447 - info: [debug] [BOOTSTRAP] [debug] Got command of type ACTION
2015-02-17 17:01:58:448 - info: [debug] [BOOTSTRAP] [debug] Got command action: pressKeyCode
2015-02-17 17:01:58:464 - info: [debug] [BOOTSTRAP] [debug] Returning result: {"value":true,"status":0}
2015-02-17 17:01:58:464 - info: [debug] Responding to client with success: {"status":0,"value":true,"sessionId":"7192f3b3-7680-4fec-9d3d-09eb2b916930"}
2015-02-17 17:01:58:466 - info: <-- POST /wd/hub/session/7192f3b3-7680-4fec-9d3d-09eb2b916930/appium/device/press_keycode 200 20.998 ms - 76 {"status":0,"value":true,"sessionId":"7192f3b3-7680-4fec-9d3d-09eb2b916930"}
```

```
2015-02-17 17:02:19:308 - info: --> POST /wd/hub/session/7192f3b3-7680-4fec-9d3d-09eb2b916930/appium/device/keyevent {"sessionId":"7192f3b3-7680-4fec-9d3d-09eb2b916930","keycode":82}
2015-02-17 17:02:19:309 - info: [debug] Pushing command to appium work queue: ["pressKeyCode",{"keycode":82,"metastate":null}]
2015-02-17 17:02:19:311 - info: [debug] [BOOTSTRAP] [debug] Got data from client: {"cmd":"action","action":"pressKeyCode","params":{"keycode":82,"metastate":null}}
2015-02-17 17:02:19:311 - info: [debug] [BOOTSTRAP] [debug] Got command of type ACTION
2015-02-17 17:02:19:311 - info: [debug] [BOOTSTRAP] [debug] Got command action: pressKeyCode
2015-02-17 17:02:19:326 - info: [debug] [BOOTSTRAP] [debug] Returning result: {"value":true,"status":0}
2015-02-17 17:02:19:326 - info: [debug] Responding to client with success: {"status":0,"value":true,"sessionId":"7192f3b3-7680-4fec-9d3d-09eb2b916930"}
2015-02-17 17:02:19:327 - info: <-- POST /wd/hub/session/7192f3b3-7680-4fec-9d3d-09eb2b916930/appium/device/keyevent 200 18.395 ms - 76 {"status":0,"value":true,"sessionId":"7192f3b3-7680-4fec-9d3d-09eb2b916930"}
```

**Comment by hectorv** [comment]
Have you been able to reproduce this? It's still happening with Selendroid 0.13.0 (through Appium 1.3.7). It works fine with Appium 1.3.3.

**Comment by hectorv** [comment]
Just noticed the following:

-- with unicodeKeyboard : True (cannot send keyevent and test fails) 
-- with unicodeKeyboard : False (sends keyevent and test passes) 

which looks similar to this past issue:
https://github.com/appium/appium/issues/3631

**Comment by jlipps** [comment]
@imurchie can you think of any reason the unicode keyboard would interact with the keyevents? is there a special check we need here?

**Comment by imurchie** [comment]
I don't see any reason for it to interact. In Selendroid mode the keycode press uses `adb`.

When I run (with master, and the latest from npm) a test using Selendroid with `unicodeKeyboard` on the keycode call leaves this log:

```shell
info: --> POST /wd/hub/session/de5ccc70-5dc7-42d6-b7e4-a60c19660203/appium/device/keyevent {"keycode":82}
info: [debug] executing cmd: /Users/isaac/code/adt-bundle/sdk/platform-tools/adb -s emulator-5554 shell "input keyevent 82"
info: [debug] Responding to client with success: {"status":0,"value":null,"sessionId":"de5ccc70-5dc7-42d6-b7e4-a60c19660203"}
info: <-- POST /wd/hub/session/de5ccc70-5dc7-42d6-b7e4-a60c19660203/appium/device/keyevent 200 2134.039 ms - 76 {"status":0,"value":null,"sessionId":"de5ccc70-5dc7-42d6-b7e4-a60c19660203"}
```

**Comment by jlipps** [comment]
@imurchie your PR has been merged, should this issue be closed or updated?

**Comment by imurchie** [comment]
I would say it should remain open. I think @hectorv is doing something wrong, and it would be good to clear it up.

**Comment by hectorv** [comment]
@imurchie @jlipps thanks for looking into this!
I hacked this minimal script that you can use to reproduce this issue consistently (you just need to modify the global variables accordingly):
https://gist.github.com/hectorv/6d9e7b614b1b8f2beb73

**Comment by jlipps** [comment]
@imurchie i took a look at the gist and can't see anything obviously off. can you?

**Comment by imurchie** [comment]
This ought to be fixed with the latest unicode ime.

**Comment by hectorv** [comment]
This is now fixed in Appium `v1.4.0`, thanks!
